docker-compose.yml

```diff
@@ -1,16 +1,15 @@
-version: "3.8"
```

Comment:
Does single and double quotes have a difference on windows? 👀

Reply:
Idts, fixed to double quotes